packages/eas-json/src/submit/schema.ts

```diff
@@ -26,6 +26,7 @@ export const IosSubmitProfileSchema = Joi.object({
   companyName: Joi.string(),
   appName: Joi.string(),
   bundleIdentifier: Joi.string(),
+  meta: Joi.string(),
```

Comment:
what is meta?
`metadata` or maybe `storeMetadata` if we want to keep consistent naming for android too, or `appstoreMetadata` if we don't
if it's a path maybe also add `File` or `Path` suffix to the name

Reply:
Agreed with Wojtek, `meta` is really confusing. I could think it's somehow related to Meta (formerly Facebook).

Reply:
Alright, I think both `metadata` or `storeMetadata` (or even `storeConfig`) should be best here. The store config file should be able to support other stores in the future too. So I would avoid `appStoreMetadata`.